client/src/components/Messages.jsx

```diff
@@ -150,15 +161,15 @@ function Messages() {
           && (
           <Grid item xs={9}>
             <List style={{ height: '70vh', overflowY: 'auto' }}>
-              {messages.map((message, index) => (
-                <Message message={message} key={index.toString()} userId={currentUser.uid} />
+              {messages.map((currentMessage, index) => (
+                <Message message={currentMessage} key={index.toString()} userId={currentUser.uid} />
```

Comment:
Here to avoid using the .toString() method I think we could consider giving each currentMessage object an id. I think we just need to make a small change to the context/index.js file